test/system/700-play.bats

```diff
@@ -278,3 +278,46 @@ status: {}
     run_podman 125 play kube - < $PODMAN_TMPDIR/test.yaml
     assert "$output" =~ "invalid annotation \"test\"=\"$RANDOMSTRING\"" "Expected to fail with annotation length greater than 63"
 }
+
+@test "podman play kube - default log driver" {
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+
+    #Config with journald as default
+    containersConf=$PODMAN_TMPDIR/containers.conf
```

Comment:
I preferred the version of the test before. IMHO it's sufficient to just test whatever the default is. This way, we don't have to worry about it.

Reply:
The reported problem is that journald is default but play kube uses file.
Now on a system with file as default this test would not catch the bug if you just compare default.
The test must ensure that it will work with both file and journald as default.

Reply:
Not exactly.  The problem in detail was that the default log driver for `play kube` was left empty which in turn chose the file driver.  Note that the driver "type" was left blank/empty as well.

The test would not work on previous Podman version since the default logger from `podman info` would return k8s-file while the driver type would be `""`.

Reply:
If (big If) I understand the test correctly, `skip_if_journald_unavailable` would be a good idea. Looks like you need `skip_if_remote` too.

Reply:
It should work for `remote` as well.  There's probably another fart somewhere that needs wiring.  I _guess_ that remote sets the log driver to `""` so we probably need to change the handler to handle that case and let `""` use the default log driver.